data/core/macros/abilities.cfg

```diff
@@ -227,6 +227,8 @@ All adjacent lower-level units from the same side deal 25% more damage for each
 #define ABILITY_ILLUMINATES
     # Canned definition of the Illuminates ability to be included in an
     # [abilities] clause.
+#arg HALO
+""#endarg
```

Comment:
@CelticMinstrel do you have any opinion on this being an argument for the macro? Technically speaking I assume it's no problem, but this would be the only ability in this file that takes any arguments.

Reply:
in weapon-special.cfg , there is a special plague which also takes an argument (even if I have never seen it used in the mainline)